report/report-generator.js

```diff
@@ -102,13 +93,58 @@ class ReportGenerator {
     return '<script src="/pages/scripts/lighthouse-report.js"></script>';
   }
 
+  _createSections(aggregations) {
+    return aggregations.reduce((menu, aggregation) => {
+      if (menu.indexOf(aggregation.type.name) === -1) {
+        menu.push(aggregation.type.name);
+      }
+      return menu;
+    }, []);
+  }
+
+  _createPWAAuditsByTag(aggregations) {
```

Comment:
though, should this be pulling from across aggregations? Feel free to defer on this, but should PWAness be just another aggregation that is specially recognized by the reporter? Doing it this way feels like our aggregation layer doesn't do everything we want, so this is hacked on top